src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/FinancialTransactionCreateDTO.java

```diff
@@ -11,6 +11,7 @@
 
 public record FinancialTransactionCreateDTO(@Min(1) @NotNull Long walletId,
                                             @Digits(integer = 13, fraction = 2) @PositiveOrZero BigDecimal amount,
-                                            String description, Instant date,
+                                            @NotNull String description,
```

Comment:
Why description has not `@NotNull` validation? This is optional field.
Also UpdateDTO for transactions has validation for date time format, probably should also be added here too. And UpdateDTO is missing also `@NotNull` validation